### PR TITLE
Vagrant instructions

### PR DESCRIPTION
**Pull request by kreczko**
Almost forgot about this one. This pull request contains:
 - How to build your own vagrant box (https://www.vagrantup.com/)
 - How to use the existing box (kreczko/cmssw)
 - Added all the repos that are needed

### CODE REVIEW

**Comment by kreczko** [comment]
bump

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+https://twiki.cern.ch/twiki/bin/view/CMSPublic/SDTCMSSW_aptinstaller
+
```
Comment (EmyrClement) [inline README.md]: What's this link for?

File: README.md [hunk 2]
```diff
@@ -65,4 +67,106 @@ nohup cmsRun BristolAnalysis/NTupleTools/test/makeBLT_cfg.py CMSSW=53X centreOfM
...
 ## Bugs
-Please report any problems on our [issues page](https://github.com/BristolTopGroup/NTupleProduction/issues).
\ No newline at end of file
+Please report any problems on our [issues page](https://github.com/BristolTopGroup/NTupleProduction/issues).
+
+# Vagrant
+
+## How to use the existing CMSSW box
+In case you are happy with the existing CMSSW box, feel free to use it.
+
```
Comment (EmyrClement) [inline README.md]: What's the "existing CMSSW box" ?

File: README.md [hunk 3]
```diff
@@ -65,4 +67,106 @@ nohup cmsRun BristolAnalysis/NTupleTools/test/makeBLT_cfg.py CMSSW=53X centreOfM
...
+```shell
+# First download it onto your machine
+wget <path>
```
Comment (EmyrClement) [inline README.md]: What should "<path>" be ?

File: README.md [hunk 4]
```diff
@@ -65,4 +67,106 @@ nohup cmsRun BristolAnalysis/NTupleTools/test/makeBLT_cfg.py CMSSW=53X centreOfM
...
+# install repositories
+sudo cp vagrant/cvmfs.repo /etc/yum.repos.d/.
+sudo wget -O /etc/pki/rpm-gpg/RPM-GPG-KEY-CernVM http://cvmrepo.web.cern.ch/cvmrepo/yum/RPM-GPG-KEY-CernVM
+# install CVMFS and dependencies
+sudo yum install cvmfs freetype freetype-devel -y
+# configure CVMFS
+sudo cp vagrant/default.local /etc/cvmfs/.
+sudo cvmfs_config chksetup
+# restart autofs service
+sudo service autofs restart
```
Comment (EmyrClement) [inline README.md]: These lines aren't formatted very well (on the same line).

File: README.md [hunk 5]
```diff
@@ -65,4 +67,106 @@ nohup cmsRun BristolAnalysis/NTupleTools/test/makeBLT_cfg.py CMSSW=53X centreOfM
 More information can be found at [Bristol Ntuple Recipes twiki page](https://twiki.cern.ch/twiki/bin/view/CMS/BristolNTuplerRecipes), although it's outdated.
 
 ## Bugs
-Please report any problems on our [issues page](https://github.com/BristolTopGroup/NTupleProduction/issues).
\ No newline at end of file
+Please report any problems on our [issues page](https://github.com/BristolTopGroup/NTupleProduction/issues).
+
+# Vagrant
+
```
Comment (EmyrClement) [inline README.md]: I assume you have to download vagrant?

Comment (kreczko) [inline README.md]: Yes.

File: README.md (hunk 1 above)
Comment (kreczko) [inline README.md]: Right, forgot to remove. The initial idea was to do it on Ubuntu (and therefore travis)

File: README.md (hunk 2 above)
Comment (kreczko) [inline README.md]: kreczko/cmssw

File: README.md (hunk 5 above)
Comment (EmyrClement) [inline README.md]: Can you provide a link?

File: README.md (hunk 3 above)
Comment (kreczko) [inline README.md]: https://copy.com/bmAv9ng0wVSlKw4G

File: README.md (hunk 4 above)
Comment (kreczko) [inline README.md]: will make them look nicer

Comment (kreczko) [inline README.md]: done

**Pull request closed without merging**